### PR TITLE
Add swap limit fix page

### PR DESCRIPTION
**Pull request by dmytrotkk**
In this PR:

- Added a new page with instructions for node swap limit fix

### CODE REVIEW

**Review by DmytroNazarenko — APPROVED** [review]

**Review by badrogger — APPROVED** [review]

File: src/content/docs/ecosystem/validators/swap-limit-fix.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+If you see the following line at the end of the output, it means the swap limiting capability is disabled:
+
+```
+WARNING: No swap limit support
+```
+
+## Fixing the Swap Limit Issue
+
+> **Note:** If you will encounter with any issues during the following steps, please stop and contact SKALE team.
```
[the remark this answers is not in the document]
Comment (dmytrotkk) [reply src/content/docs/ecosystem/validators/swap-limit-fix.mdx:50]: addressed

File: src/content/docs/ecosystem/validators/swap-limit-fix.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+title: SKALE Node Swap Limit Fix
+description: SKALE Node Swap Limit Fix for validators
+---
+
```
[the remark this answers is not in the document]
Comment (dmytrotkk) [reply src/content/docs/ecosystem/validators/swap-limit-fix.mdx:5]: addressed

File: src/content/docs/ecosystem/validators/swap-limit-fix.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+```
+WARNING: No swap limit support
+```
+
+## Fixing the Swap Limit Issue
+
+> **Note:** If you will encounter with any issues during the following steps, please stop and contact SKALE team.  
+
+> **Warning:** If your node is running on Google Cloud, commands may differ. Please check discussions on [Stack Exchange](https://unix.stackexchange.com/a/495058) or contact SKALE team.
```
[the remark this answers is not in the document]
Comment (dmytrotkk) [reply src/content/docs/ecosystem/validators/swap-limit-fix.mdx:52]: addressed

File: src/content/docs/ecosystem/validators/swap-limit-fix.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Follow these steps to enable swap limiting capability:
+
```
[the remark this answers is not in the document]
Comment (dmytrotkk) [reply src/content/docs/ecosystem/validators/swap-limit-fix.mdx:55]: addressed

**Pull request merged**